proto/cmp/types/v1alpha/traveller.proto

```diff
@@ -46,8 +45,11 @@ message ExtensiveTraveller {
   // referencing services linked to specific participants, like baggage.
   int32 traveller_id = 1;
 
+  // Traveller type from enum below
+  TravellerType type = 2;
+
```

Comment:
Travellertype was not included on purpose to avoid contradictionary information between validate and mint

Reply:
Removed after the discussion on Discord. (Reason is we want to separate the those fields between basic and extensive traveller messages so the buyer does not change the traveller type in the mint request and only reference to the basic traveller by using the same id.)